scripts/provisioning/templates/s3.config.tmpl.1-node

```diff
@@ -38,6 +38,11 @@ cortx:
         max_units : "8"
       service:
         instances : "1"
+    s3bgdelete:
```

Comment:
add this in to sample file as well. 
if applicable then 3 node tmpl and sample file as well .

Reply:
also, need to add above parameters in to init tmplate and sample file

Reply:
done